rats-tls/dist/rpm/rats_tls.spec.in

```diff
@@ -37,6 +37,30 @@ ExclusiveArch: x86_64
 %description
 rats-tls is a protocol to establish secure and trusted channel by integrating enclave attestation with transport layer security.
 
+%package host
+Summary: host mode package for rats-tls
+
+%description host
+rats-tls-host is the "host" build mode package of rats-tls.
+
+%package occlum
+Summary: occlum mode package for rats-tls
+
+%description occlum
+rats-tls-occlum is the "occlum" build mode package of rats-tls.
+
+%package sgx
+Summary: sgx mode package for rats-tls
+
+%description sgx
+rats-tls-sgx is the "sgx" build mode package of rats-tls.
+
+%package tdx
+Summary: tdx mode package for rats-tls
+
+%description tdx
```

Comment:
Please use `Conflicts` in spec to indicate that those packages will not be installed at the same time